WikiDrips/WikiTableViewController.swift

```diff
@@ -33,13 +36,14 @@ class WikiTableViewController: UITableViewController {
     }
     
     override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
-        let cell = tableView.dequeueReusableCell(withIdentifier: "WikiDoc", for: indexPath) as! WikiTableViewCell
-        let wikiDoc = wikiDocs[indexPath.section][indexPath.row]
-        cell.wikiTitleLabel?.text = wikiDoc.title
-        cell.wikiDateLabel?.text = wikiDoc.editedDate()
-//        cell.wikiTitleImage?.image = wikiDoc.imageInitials // TODO: Generate UIImage
-
-        return cell
+        let cell = tableView.dequeueReusableCell(withIdentifier: "WikiDoc", for: indexPath)
+        if let wikiCell = cell as? WikiTableViewCell {
+            let wikiDoc = wikiDocs[indexPath.section][indexPath.row]
+            wikiCell.wikiTitleLabel?.text = wikiDoc.title
+            wikiCell.wikiDateLabel?.text = dateFormatter.string(from: wikiDoc.pubDate)
+//            cell.wikiTitleImage?.image = wikiDoc.imageInitials // TODO: Generate UIImage
+        }
+        return cell        
     }
```

Comment:
So a typical pattern is to use `guard` statements to check your assumptions up front, this pattern will avoid having a bunch of nested `if` statements when there are a lot of conditions you need to be true. It also kinda separates your bad path from your golden path. In this case for example you could do something like:

```
    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
        guard let wikiCell = tableView.dequeueReusableCell(withIdentifier: "WikiDoc", for: indexPath) as? WikiTableViewCell else {
            // Log or ignore casting failure
            return UITableViewCell()
        }
        
        let wikiDoc = wikiDocs[indexPath.section][indexPath.row]
        wikiCell.wikiTitleLabel?.text = wikiDoc.title
        wikiCell.wikiDateLabel?.text = dateFormatter.string(from: wikiDoc.pubDate)
//        cell.wikiTitleImage?.image = wikiDoc.imageInitials // TODO: Generate UIImage
        
        return wikiCell
    }
```